src/main/groovy/com/athaydes/spockframework/report/template/TemplateReportCreator.groovy

```diff
@@ -79,7 +79,7 @@ class TemplateReportCreator implements IReportCreator {
 
     def createFeaturesCallback( SpecData data ) {
         return [ eachFeature: { Closure callback ->
-            for ( feature in data.info.allFeatures ) {
+            for ( feature in data.info.allFeaturesInExecutionOrder ) {
```

Comment:
oh, didn't know this method. Maybe we should use this everywhere instead of the shorter one.